saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationApp.scala

```diff
@@ -112,8 +112,8 @@ object EntityRelationApp extends Logging {
     val testTokens = tokens.getTestingInstances.toSet.toList
 
     // load pre-trained independent models
-    ClassifierUtils.LoadClassifier(jarModelPath, PersonClassifier, OrganizationClassifier, LocationClassifier,
-      WorksForClassifier, LivesInClassifier, LocatedInClassifier, OrgBasedInClassifier)
+    //    ClassifierUtils.LoadClassifier(jarModelPath, PersonClassifier, OrganizationClassifier, LocationClassifier,
+    //      WorksForClassifier, LivesInClassifier, LocatedInClassifier, OrgBasedInClassifier)
```

Comment:
Right; do you intent to keep it this way? 
How we about we uncomment these lines, and add note above it saying "The following lines (pre-training) are not necessary, although without pre-training the performance might drop."